### DISCUSSION TITLE
Quotas in Lambda Test Tool

### DISCUSSION THREAD

**Discussion by schmallaria** [aws/aws-lambda-dotnet#1810] [Q&A] [answered]
Hi,

as you all know, there are some quotas which applies to Lambda function execution, e.g. max. 6 MB for request and response.
Can Lambda Test Tool simulate such quotas? At the moment it seems not.. 

Regards,
Christian

**Comment by ashishdhingra** [comment, marked as answer]
> Hi,
> 
> as you all know, there are some quotas which applies to Lambda function execution, e.g. max. 6 MB for request and response. Can Lambda Test Tool simulate such quotas? At the moment it seems not..
> 
> Regards, Christian

@schmallaria Good morning. Thanks for the question. Quotas are imposed by Lambda service. Lambda Test tool just allows you to test your lambda functions in your local environment. Hence, quotas are not supported. There is an existing `p3` backlogged issue https://github.com/aws/aws-lambda-dotnet/issues/1247 to mimic the Lambda life cycle in test tool. But it's unsure if we could support quotas and there is no definitive ETA for it's implementation.

Thanks,
Ashish
